src/shared/proto/tbots_frankie_status.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto2";
+
+message Tbots_EncoderData {
+//TODO define encoder data and create filter for it
+}
+
+message Tbots_FrankieStatus {
```

Comment:
We can probably just make this `TbotsRobotStatus`

Reply:
If we're going with the oneof structure then the statuses do need to versioned, i.e. 2014, etc. for a hybrid fleet, say